server/__init__.py

```diff
@@ -98,3 +131,8 @@ class Root(object):
     info['apiRoot'].image = api.ImageResource()
     info['apiRoot'].segmentation = api.SegmentationResource()
     info['apiRoot'].study = api.StudyResource()
+
+    # Move girder app to /girder, serve isic_archive app from /
+    info['serverRoot'], info['serverRoot'].girder = (Webroot(),
+                                                     info['serverRoot'])
+    info['serverRoot'].api = info['serverRoot'].girder.api
```

Comment:
We're not ready to remap the API yet, since we still have Angular code where it's hardcoded. Comment out this line.